### ISSUE TITLE
for installed add-ons the toggle animates to on when opening disco pane

### ISSUE THREAD

**Issue by designakt** [mozilla/addons-frontend#898]
If an add-on is already installed when disco pane is launched, the toggle animates to on.
This might be misunderstood as this add-on was just turned on. (Which might be creepy on opening disco pane.)
![discopaneimprovement-toggleanimatedonlaunch](https://cloud.githubusercontent.com/assets/8145891/17517632/8221bd0a-5e44-11e6-903c-df4a7d9d10b8.gif)

Expected behavior: The toggle should be in its position on launch.

**Comment by andymckay** [comment]
When disco pane opens we don't know what add-ons are installed. We have to query each add-on to find out if its installed. 

Should we not show the button until we know or should there be an intemediary state?

**Comment by designakt** [comment]
Not having an animation in that situation would already improve this a lot.
In my recording frame 2 already shows the ✔️ check mark which suggests to me that we already know that it is installed.
![discopaneimprovement-toggleanimatedonlaunch-frame2](https://cloud.githubusercontent.com/assets/8145891/17523357/936d9eb0-5e5a-11e6-95c4-6f7cfcd075a8.gif)
After that it is only the slide animation and transition to green.

(I did talk to @muffinresearch about that before, and we had found a solution. Not sure what happened then. I couldn't find the issue for it.)
